typewise_alert.test.py

```diff
@@ -5,6 +5,9 @@
 class TypewiseTest(unittest.TestCase):
   def test_infers_breach_as_per_limits(self):
     self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
+    self.assertTrue(typewise_alert.check_and_alert('TO_CONTROLLER', 'PASSIVE_COOLING', 20) == 'NORMAL')
```

Comment:
This assert states that check_and_alert converts 3 parameters into one return value string.
If you had to re-state its actual functionality as a data-conversion, what would you state?